src/hashFile.ts

```diff
@@ -0,0 +1,13 @@
+import { crypto } from "./deps.ts";
+
+export const hashFile = (url: string): string => {
```

Comment:
I'd actually love if the hash changed to be something like `filename-[smallhash].ext`. Would make debugging a lot easier.

Reply:
I've done this

Reply:
love this